src/rpc_api/mod.rs

```diff
@@ -190,6 +192,14 @@ pub mod chain_api {
     pub const CHAIN_SET_HEAD: &str = "Filecoin.ChainSetHead";
     pub const CHAIN_GET_MIN_BASE_FEE: &str = "Filecoin.ChainGetMinBaseFee";
     pub const CHAIN_GET_MESSAGES_IN_TIPSET: &str = "Filecoin.ChainGetMessagesInTipset";
+
+    #[derive(Serialize, Deserialize)]
+    #[serde(rename_all = "PascalCase")]
+    pub struct MessageInTipset {
+        pub cid: LotusJson<Cid>,
+        pub message: LotusJson<crate::shim::message::Message>,
+    }
```

Comment:
This is `ApiMessage`.

In general, we don't want to directly use `LotusJson` types in our API. When using our client RPC library, one should never have to unpack `LotusJson`.

Reply:
Fixed